stdpopsim/utils.py

```diff
@@ -268,3 +268,43 @@ def clip_and_shift_intervals(intervals, left, right):
         warnings.warn(f"No intervals remain after clipping to [{left}, {right}]")
     intervals -= left
     return intervals
+
+
+def parse_population_sample_pairs(pop_sample_pairs):
+    """
+    Parse a list of strings of the form "population_name:integer" into dict
+    {population_name:integer}.
+
+    FIXME: For back-compatibility, return a list of integers if the
```

Comment:
This isn't really a FIXME, more something that we probably want to do at some point but not quite sure when. 
```suggestion
    For back-compatibility, return a list of integers if the
```